apps/android/LiveVideoApp/app/src/debug/java/com/twilio/livevideo/app/ExtensionUtil.kt

```diff
@@ -0,0 +1,43 @@
+package com.twilio.livevideo.app
+
+import android.annotation.SuppressLint
+import android.content.ComponentName
+import android.content.Intent
+import android.os.Bundle
+import androidx.annotation.StyleRes
+import androidx.core.util.Preconditions
+import androidx.fragment.app.Fragment
+import androidx.fragment.testing.R
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+
+@SuppressLint("RestrictedApi")
+inline fun <reified T : Fragment> launchFragmentInHiltContainer(
```

Comment:
This class is a workaround to make it `Hilt` testing works with fragments. By default, all the fragments requires a container (activity), but it's not provided by the framework.